target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -357,6 +357,11 @@ ramips_setup_macs()
 		lan_mac=$(mtd_get_mac_binary factory_info 13)
 		wan_mac=$(macaddr_add "$lan_mac" 1)
 		;;
+	miwifi-mini)
+		wan_mac=$(cat /sys/class/net/eth0/address)
+		lan_mac=$(cat /sys/class/net/eth0/address)
+		lan_mac=$(macaddr_setbit_la "$lan_mac")
+		;;
```

Comment:
you can do this in 2 lines

+		wan_mac=$(cat /sys/class/net/eth0/address)
+              lan_mac=$(macaddr_setbit_la "$wan_mac")